### PR TITLE
Add func for interface test

### PR DESCRIPTION
**Pull request by kylazhang**
1. Add property for interface xml
2. Update get_iface_by_mac by getting more values for iface
3. Add new func for vm iface modify and vm boot order change

### CODE REVIEW

File: virttest/utils_test/libvirt.py [hunk 1]
```diff
@@ -3174,3 +3174,95 @@ def create_secret(params):
...
 
     return sec_uuid
+
+
+def modify_vm_iface(vm_name, oper, iface_dict, index=0):
+    """
+    Modify interface xml and do operations
+
+    :param oper: Operation need to do, current we have 2 choice
+    1. "update_iface": modify iface according to dict and updated in vm
```
Comment (chunfuwen) [inline virttest/utils_test/libvirt.py]: please sync doc string with actual parameters

File: virttest/utils_test/libvirt.py [hunk 2]
```diff
@@ -3174,3 +3174,95 @@ def create_secret(params):
...
+        return iface.xml
+
+
+def change_boot_order(vm_name, device_tag, boot_order, index=0):
+    """
+    Change the order for disk/interface device
+
+    :param vm_name: name of vm
+    :param device_tag: the device tag in xml
+    :param index: which device to change
```
Comment (chunfuwen) [inline virttest/utils_test/libvirt.py]: miss doc string for boot_order

**Comment by lento-sun** [comment]
@kylazhang now this pr has conflicts and cannot be used in our CI jobs, which cause some acceptance cases failed. pls have a check.

**Comment by kylazhang** [comment]
@lento-sun @chunfuwen please help to check and merge it asap as it related to interface xml and can cause conflict time to time, thanks

**Comment by lento-sun** [comment]
LGTM

**Comment by lento-sun** [comment]
LGTM and the all related cases passed in our acceptance test job

**Comment by kylazhang** [comment]
@chunfuwen please help to review and merge it as it also affect others patches, and also the acceptance job related cases passed

**Pull request merged**